content/spin/install.md

```diff
@@ -22,7 +22,11 @@ Spin runs on Linux (amd64 and arm64), macOS (Intel and Apple Silicon), and Windo
 
 {{ startTab "Linux"}}
 
-There are multiple ways to install Spin. The easiest is to use the installer script, hosted on this site. The script installs Spin along with a starter set of language templates and plugins:
+You can now manage your Spin installation via [Homebrew](https://brew.sh/). Homebrew automatically installs Spin templates and Spin plugins, and on uninstall, will prompt you to delete the directory where the templates and plugins were downloaded:
+
+<pre class="bash spin-install" id="spin-install-linux"><code>$ brew tap fermyon/tap && brew install fermyon/tap/spin</code></pre>
```

Comment:
I'd recommend breaking this into two steps to emphasize that they first need to install the Fermyon tap, a repository of all the Fermyon Homebrew packages, and then isntall spin with `fermyon/tap/spin`. See Hashicorp for example: https://developer.hashicorp.com/nomad/docs/install.

We also should add an instruction on how to upgrade to latest:
`brew upgrade fermyon/tap/nomad`

Reply:
Again, I agree about documenting the upgrade command, but with the current docs structure this would go better on https://developer.fermyon.com/spin/upgrade

Reply:
Please see new PR that updates the upgrade page
https://github.com/fermyon/developer/pull/827